### ISSUE TITLE
Doesn't work with new version 11 / Atmosphere

### ISSUE THREAD

**Issue by purzer0** [retronx-team/mtp-server-nx#16]
Please update

**Comment by sanio11** [comment]
Confirm, not work on pre-release Atmosphere.
Pre-release

**Comment by dezem** [comment]
You can temporary use my forked version with fw11 support

https://github.com/dezem/mtp-server-nx/releases/tag/1.4

**Comment by m4xw** [comment]
Do ppl even use this anymore

**Comment by dezem** [comment]
Yes, of course, this is the fastest way to transfer data from the pc to the switch or vice versa.
Without having to reboot them to hekate or removing the sd...

**Comment by LiloBzH** [comment]
That's very very slow :(
Is it normal ?

**Comment by dezem** [comment]
Ya it's normal...use hekate UMS instead for better performance
